bin/resin-preload

```diff
@@ -2,9 +2,12 @@
 
 'use strict'
 
-const fs = require('fs')
+const Docker = require('dockerode')
 const preload = require('..')
 const info = require('../package.json')
+const streamToPromise = require('stream-to-promise')
+
+const docker = new Docker({Promise})
```

Comment:
This will only ever use the local docker, which I assume is fine for this use-case, just thought i'd let you know.

That is also assuming that the default is `{ socketPath: '/var/run/docker.sock' }` and whatever the equivalent on windows is, it may be worth adding this in anyway.